README.md

```diff
@@ -1,6 +1,6 @@
 # Alex Kirilenko . Jarvis Consulting
 
- I graduated with MSc in Electrical Engineering from the University of Saskatchewan. During my graduate studies I focused on scientific computing, numerical methods and applied mathematics. Recently, I have joined Jarvis as Software Developer / Data Engineer where I learn the agile software development lifecycle practices and get hands-on experience with a broad stack of backend and data engineering technologies. I strive to deliver software products of the highest quality that satisfy the client's needs, and can be easily maintained by other developers. Overall, I am excited to work on challenging projects and to contribute to their success as a part of a team. 
+ I graduated with MSc in Electrical Engineering from the University of Saskatchewan. During my graduate studies, I focused on scientific computing, numerical methods and applied mathematics. Recently, I have joined Jarvis as Software Developer / Data Engineer where I learn the agile software development lifecycle practices and get hands-on experience with a broad stack of backend and data engineering technologies (including Spring Boot, Hibernate, REST API, etc.). I strive to deliver software products of the highest quality that satisfy the client's needs, and can be easily maintained by other developers. Overall, I am excited to work on challenging projects and to contribute to their success as a part of a team. 
```

Comment:
I fixed some minor grammar for you.


I graduated with MSc in Electrical Engineering from the University of Saskatchewan. During my graduate studies, I focused on scientific computing, numerical methods and applied mathematics. I recently joined Jarvis as a software developer/data engineer. I have learned the agile software development lifecycle practices and get hands-on experience with a broad stack of back-end and data engineering technologies (including Spring Boot, Hibernate, REST API, etc.). I strive to deliver software products of the highest quality that satisfy the client's needs and easily be maintained by other developers. Overall, I am excited to work on challenging projects and to contribute to their success as a part of a team.